bazel/repositories.bzl

```diff
@@ -133,33 +133,101 @@ def _envoy_api_deps():
     api_bind_targets = [
         "address",
         "base",
-        "bootstrap",
-        "discovery",
-        "cds",
-        "discovery",
-        "eds",
         "grpc_service",
         "health_check",
-        "lds",
-        "metrics",
         "protocol",
+    ]
+    for t in api_bind_targets:
+        native.bind(
+            name = "envoy_" + t,
+            actual = "@envoy_api//envoy/api/v2:" + t + "_cc",
+        )
+    native.bind(
+        name = "envoy_bootstrap",
+        actual = "@envoy_api//envoy/config/bootstrap/v2:bootstrap_cc",
+    )
+    native.bind(
+        name = "envoy_metrics",
+        actual = "@envoy_api//envoy/service/metrics/v2:metrics_service_cc",
+    )
+    native.bind(
+        name = "envoy_load_stats",
+        actual = "@envoy_api//envoy/service/load_stats/v2:lrs_cc",
+    )
+    native.bind(
+        name = "envoy_ratelimit",
+        actual = "@envoy_api//envoy/config/ratelimit/v2:rls_cc",
+    )
+    native.bind(
+        name = "envoy_accesslog",
+        actual = "@envoy_api//envoy/service/accesslog/v2:als_cc",
+    )
+    native.bind(
```

Comment:
I'd be happy to clean this up separately to avoid binding so many names...

Reply:
+1, yes please

Reply:
I would prefer not using binds (I think they might be an antipattern as they make working with multiple workspaces difficult). We should use "actual" names, since the path is more or less stable.

Reply:
So, yeah, some history here. When we first started to Bazelify Envoy, it was unclear to me how best to handle external deps from a few perspectives:
* How would we be able to map to prebuilts at Lyft as well as use native Bazel repositories?
* How would we redirect dependencies when we imported into Google's monorepo.

The prebuilt situation has stabilized and doesn't apply here (we don't prebuild `@envoy_api`). The import for Google is also now moot. While it's a little neater to rebind, TBH we're sed-ing a ton during the import, so rewriting `@envoy_api` is trivial.

So, definitely for this dependency, which is now sprawling with binds, let's just switch to direct reference to @envoy_api everywhere.

Reply:
+1 if we can just do this now since this is already a giant change and mostly find/replace. I wondered why we did this in the first place, thanks for the explanation.

Reply:
Done, with a sed swoosh